examples/micro-rdk-esp32-sensor-examples/README.md

```diff
@@ -1,143 +0,0 @@
-# Viam Micro-RDK ESP32 Sensor Modules Example
```

Comment:
I think this would be the README.md to keep, since it has content, unlike the one from micro-rdk-modular-driver-examples.